components/validator/introduction.rst

```diff
@@ -0,0 +1,86 @@
+.. index::
+   single: Validator
+   single: Components; Validator
+
+The Validator Component
+=======================
+
+    The Validator component provides tools to validate values following the
+    `JSR-303 Bean Validation specification`_. With the component, this is done in two parts:
+    * ``Contraints``: a constraint describes a rule that need to be validated
+    * ``Validators``: a list of classes that implement the validation logic for common usages
+
+
+
+Installation
+------------
+
+You can install the component in 2 different ways:
+
+* :doc:`Install it via Composer </components/using_components>` (``symfony/validator`` on `Packagist`_);
+* Use the official Git repository (https://github.com/symfony/Validator).
+
+Then, require the vendor/autoload.php file to enable the autoloading mechanism provided by Composer. Otherwise, your application won't be able to find the classes of this Symfony component.
```

Comment:
Please replace this by:

```rst
.. include:: /components/require_autoload.rst.inc
```

This way, we only need to update one file when something in this paragraph needs to be changed